### ISSUE TITLE
Taxonomy - after adding & publishing a new term, the taxonomy must also be published but this isn't obvious

### ISSUE THREAD

**Issue by planetClaire** [OrchardCMS/OrchardCore#7054]
Adding terms to the taxonomy works differently in the UI to adding items to a Bag.  This is fine, but in the case of a taxonomy, it's not clear that the taxonomy is in a draft state after adding the term.  A warning message could be added.  To be clear:

1. Edit a taxonomy content item
2. Click the "Add [term type]" button (or edit an existing)
3. On the "New [term]" page, fill out the details & click "Publish"
4. When redirected back to Edit Taxonomy, there could be a warning at the top "You need to publish your changes to this taxonomy"

By the way the taxonomy feature is great. I've just been playing around with it and found the references on Autoroute and Taxonomies helpful, but it took a while to get my head around it & I'm thinking it might be nice to have a page in "How-to guides" or "Key topics" with a basic example?  I'd be happy to write something.

**Comment by sebastienros** [comment]
What about having two buttons, one Save and one Publish. Save would save the term, and publish would publish the taxonomy with the updated term. We might still want to create multiple terms before publishing the taxonomy.

TODO: Same for menu items.

**Comment by planetClaire** [comment]
Yes - as a user this is what I would expect to be the normal behaviour, thanks.
